src/command/cmd_defs.c

```diff
@@ -1719,7 +1720,9 @@ static const struct cmd_t command_defs[] = {
               { "log on|off", "Enable or disable plaintext logging of PGP encrypted messages." },
               { "log redact", "Log PGP encrypted messages, but replace the contents with [redacted]. This is the default." },
               { "char <char>", "Set the character to be displayed next to PGP encrypted messages." },
-              { "sendfile on|off", "Allow /sendfile to send unencrypted files while otherwise using PGP." })
+              { "sendfile on|off", "Allow /sendfile to send unencrypted files while otherwise using PGP." },
+              { "autoimport on|off", "Autoimport PGP keys from messages." },
```

Comment:
I'm actually not sure but maybe we should mention here "This is compatible with PSI+"? But really not sure if this is good or not. Usually things are described in a XEP but this isn't so I thought maybe it's interesting for users which other clients work with this. On the other hand I for some reason don't like to tell which other clients support something since we have to keep this information up to date..

Reply:
Well I think lets not do that. It might become outdated. Especially on some distros that updates every few decades.